pkg/controller/kogitoapp/resource/kafka.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resource
+
+import (
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure/services"
+	corev1 "k8s.io/api/core/v1"
+	"strings"
+)
+
+type kafkaProperty int
+
+const (
+	appPropKafkaBootstrapServerList kafkaProperty = iota
+
+	envVarKafkaBootstrapSuffix = "_BOOTSTRAP_SERVERS"
+)
+
+var (
+	propertiesKafkaQuarkus = map[kafkaProperty]string{
+		appPropKafkaBootstrapServerList: services.QuarkusBootstrapAppProp,
+	}
+	propertiesKafkaSpring = map[kafkaProperty]string{
+		appPropKafkaBootstrapServerList: "spring.kafka.bootstrap-servers",
```

Comment:
Shouldn't this be externalized to a constant like `services.QuarkusBootstrapAppProp` above?

Reply:
Corrected. Thanks.